### PR TITLE
Add syntax highlighting

### LINKED ISSUE #225: Markdown file missing syntax highlighting
Very small change, I noticed that some of the code blocks are missing syntax highlighting making them harder to read. The biggest example I could find was in `docs/advanced_specs.md` with:
```
import luxai_s2
custom_env_cfg = dict()
env = gym.make("LuxAI_S2-v0", collect_stats=False, **custom_env_cfg)
env.reset()
```
Would simply change to:
```python
import luxai_s2
custom_env_cfg = dict()
env = gym.make("LuxAI_S2-v0", collect_stats=False, **custom_env_cfg)
env.reset()
```

### PR DESCRIPTION
**Pull request by adamfrly**
Adding syntax highlighting to markdown console documentation.

Closes #225

### CODE REVIEW

File: kits/README.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ In each starter kit folder we give you all the tools necessary to compete. Make
...
+```Python console
 luxai-s2 kits/python/main.py kits/python/main.py -s 42 -v 2 -o replay.json
 ```
 
 To then watch the replay, upload replay.json to http://s2vis.lux-ai.org/
 
 Alternatively you can generate a openable HTML file to watch it as well by specifying the output as .html
 
-```
+```Python console
```
Comment (StoneT2000) [inline kits/README.md:17]: these should be called `python` not `Python console` I believe.

Comment (adamfrly) [reply kits/README.md:17]: I can certainly change this, but I don't think there is a difference between `python` and `python console` highlighting for this specific block of code

**Comment by StoneT2000** [comment]
@adamfrly all of the annotations should be `python` not `python-console` I believe.

File: kits/java/README.MD [hunk 2]
```diff
@@ -26,14 +26,14 @@ mvn clean compile assembly:single
...
+```Python
 luxai-s2 main.py main.py -o replay.json
 ```
 
 ## Compile for uploading to leaderboard evaluation
 
 1. Create an archive with JavaBot.jar and main.py from root dir
-    ```
+    ```Python
     tar -cvzf submission.tar.gz main.py -C localrun JavaBot.jar
```
Comment (StoneT2000) [inline kits/java/README.MD:37]: this is not python code and should not be labeled as such. And for consistency use lowercase `python`.

File: kits/java/README.MD [hunk 3]
```diff
@@ -26,14 +26,14 @@ mvn clean compile assembly:single
 2. Move jar to subfolder localrun
 3. Run with main.py
 
-```
+```Python
```
Comment (StoneT2000) [inline kits/java/README.MD:29]: same here, not python code

File: kits/rl/sb3/README.md [hunk 4]
```diff
@@ -10,13 +10,13 @@ The general structure of this RL agent is that we use a heuristic policy to hand
 
 To use the training code, run `train.py --help` for help and to train an agent run
 
-```
+```Python
```
Comment (StoneT2000) [inline kits/rl/sb3/README.md:13]: same here, not python code

**Review by StoneT2000 — CHANGES_REQUESTED** [review]
I noticed just now many of the annotations are actually on command line code, so it should not be labeled as such, please check again.

File: kits/java/README.MD (hunk 3 above)
Comment (adamfrly) [reply kits/java/README.MD]: Removed

File: kits/java/README.MD (hunk 2 above)
Comment (adamfrly) [reply kits/java/README.MD:37]: Removed

**Pull request closed without merging**